lib/hsa/hsaTrackingUpdatePhase.hpp

```diff
@@ -88,12 +91,18 @@ class hsaTrackingUpdatePhase : public hsaCommand {
     bool tracking_grab_callback(nlohmann::json& json, const uint8_t beamID);
 
 private:
-    /// Length of arrray of phases, should be 2048 x 10 x 2 for complex
+    /// Length of array of phases in bytes, should be 2048 x 10 x 2 for complex
```

Comment:
Should the dimensions be: 2048 x 12 x 2?

Reply:
Yeah `num_beams` really.